examples/image_ocr.py

```diff
@@ -88,16 +88,21 @@ def paint_text(text, w, h, rotate=False, ud=False, multi_fonts=False):
         context.paint()
         # this font list works in CentOS 7
         if multi_fonts:
-            fonts = ['Century Schoolbook', 'Courier', 'STIX', 'URW Chancery L', 'FreeMono']
-            context.select_font_face(np.random.choice(fonts), cairo.FONT_SLANT_NORMAL,
-                                     np.random.choice([cairo.FONT_WEIGHT_BOLD, cairo.FONT_WEIGHT_NORMAL]))
+            fonts = ['Century Schoolbook', 'Courier', 'STIX', 'URW Chancery L',
+                     'FreeMono']
+            context.select_font_face(np.random.choice(fonts),
+                                     cairo.FONT_SLANT_NORMAL,
```

Comment:
As a rule of thumb, if you put an arg on another line do so for each argument.
x = [super_long_a, super_long_b,super_long_c]

x = [super_long_a,
     super_long_b,
     super_long_c]